kuma/static/styles/components/compat-tables/bc-supports.scss

```diff
@@ -0,0 +1,92 @@
+/*
+Supports colouring
+********************************************************************** */
+
+.bc-supports-yes,
+.bc-supports-partial,
+.bc-supports-no,
+.bc-supports-unknown,
+.bc-table .bc-history {
+    color: $bc-color-text;
+}
+
+.bc-supports-yes a,
+.bc-supports-partial a,
+.bc-supports-no a,
+.bc-supports-unknown a {
+    color: $bc-color-text;
+    text-decoration: underline;
+}
+
+.bc-supports-yes,
+.bc-supports-yes .bc-history {
+    background-color: map-get($bc-colors, yes);
+}
+
+.bc-supports-partial,
+.bc-supports-partial .bc-history {
+    background: map-get($bc-colors, partial);
+}
+/*
+.bc-supports-partial {
+    background-image:
+        linear-gradient(to bottom right,
+            map-get($bc-colors, no) 0,
+            map-get($bc-colors, no) 4px,
+            transparent 4px,
+            transparent 65px,
+            map-get($bc-colors, no) 65px,
+            map-get($bc-colors, no) 75px,
+            transparent 75px,
+            transparent 136px,
+            map-get($bc-colors, no) 136px,
+            map-get($bc-colors, no) 146px,
+            transparent 146px
+        );
+    background-size: 100px 100px;
+    background-position: 50% 50%;
+}
+*/
```

Comment:
Will SASS strip these commented lines out of the final output CSS? If not, should probably be removed. It will always live in source control history ;p